### ISSUE TITLE
Can't select next character on test server!

### ISSUE THREAD

**Issue by BigRedBrent** [BigRedBrent/NeverwinterInvokeBot#15]
I was on the test server and noticed that the arrow keys do not allow you to select characters. If this goes live then it will remove that option for selecting the character to log into.

**Comment by madafakar** [comment]
No, you retard, in order for the arrow keys to work you have to click on the character pane.
I.e. clicking in the center of the screen activates the screen and then pressing the arrow keys gives instructions to the CENTER of the screen, which, obviously for intelligent people, does not accept such commands.

Just change the initial click to be on a character name and it will be ok.

For fuck's sake!

**Comment by BigRedBrent** [comment]
A bit harsh. I don't use the test server very much and did not even spend more than a second checking to see what I could do to circumvent the problem, because others had told me that they did not have this problem. I noticed this the second I logged into live and made appropriate changes within minutes after. I only just now read this post for the first time. But, thank you for pointing this out to me, none the less. Even if done in the most possibly juvenile way, I still appreciate it.
